myhpi/settings.py

```diff
@@ -30,9 +30,16 @@
 # Application definition
 
 INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
     "myhpi.core",
```

Comment:
I would prefer having our apps at the bottom to find them more quickly

Reply:
Ordered it now like this: First: All external apps in alphabetical order, followed by our apps in alphabetical order